src/helpers/figures/charts/chart_factory.ts

```diff
@@ -184,6 +187,18 @@ export function getSmartChartDefinition(zone: Zone, getters: Getters): ChartDefi
       legendPosition: newLegendPos,
     };
   }
+  const _dataSets = createDataSets(getters, dataSets, sheetId, dataSetsHaveTitle);
+  if (singleColumn && getData(getters, _dataSets[0]).every((e) => typeof e === "string")) {
```

Comment:
That's for the point 1) in the spec right ? It doesn't seems to work
![image](https://github.com/odoo/o-spreadsheet/assets/31045223/f0a73881-18af-4d84-9ff4-d4e08d0aebfc)

Reply:
=this part is working as intended, else you wont have a pie chart in your case :) But it looks like the data is not correctly rendered

Reply:
Ok, found the issue :) Looks like I always tested with three different values at the first 3 cells when plotting my chart :D